experimental/dds/tree2/src/feature-libraries/schemaBuilder.ts

```diff
@@ -51,25 +67,29 @@ export class SchemaBuilder<
 	 * Reduced type safety is a side effect of a workaround for a TypeScript limitation.
 	 *
 	 * See note on RecursiveTreeSchema for details.
+	 *
+	 * TODO: Make this work with ImplicitFieldSchema.
```

Comment:
Nit
```suggestion
	 * @privateRemarks TODO: Make this work with ImplicitFieldSchema.
```

Reply:
\+ the below methods

Reply:
This TODO is something callers of this method probably want to know about (specifically that it hasn't been done, and that it might get done). This is doubly true since the callers don't get type safety for this since we can't have extends clauses. Therefor making this part of the API docs was intentional.

Reply:
Regular `@remarks` then?

Reply:
I think making it very visible is important since if you don't know this and use this API, you can get really nasty errors elsewhere in your program (like when you try and use the schema), so I'm erroring on the side of making it part of the top level description. Once we determine if this is even possible to fix and/or settle on a different pattern for recursive types, we can convert this situation into more clear docs.